packages/core/src/app/Player.ts

```diff
@@ -338,12 +338,14 @@ export class Player {
     }
 
     // Pause if reached the end or the range is 0
+    // Seek to the beginning
     if (
       (!state.loop && this.finished && !state.paused && state.seek < 0) ||
       this.endFrame === this.startFrame
     ) {
       this.togglePlayback(false);
       state.paused = true;
+      state.seek = this.startFrame;
```

Comment:
This will cause endless seeking when the end frame is the same as the start frame.
As is the case in an empty animation, for example.

Reply:
It would seem that it will cause endless seeking, but the player will remain paused once it seeks to the beginning in disable loop mode. Hence, unlike the loop situation- where the player still remains playing upon reaching the end, this will pause on reaching the start frame and will not seek further.

On unit testing for the case of an empty animation, it behaves the same way as before.  It does not seek further as it remains paused.

If this is a wrong interpretation, please let me know how to correct it

Reply:
I'm not sure what unit test you're referring to, but if you edit `packages/template/src/scenes/example.tsx` to contain an empty scene with no animations:
```tsx
import {makeScene2D} from '@motion-canvas/2d';

export default makeScene2D(function* (view) {
  // do nothing
});
```
...and then run `npm run template:dev` to test your changes, you will see a "seek time" message being logged over and over again. This means that the player loop continously performs seeking for no reason:
https://github.com/motion-canvas/motion-canvas/blob/main/packages/core/src/app/Player.ts#L378-L383

Reply:
Updated based on your feedback. I've added a check in case of an empty scene before updating seek. 
The `seek time` is now being logged as expected.